specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-03-31/cdnwebapplicationfirewall.json

```diff
@@ -463,10 +463,13 @@
         "defaultCustomBlockResponseStatusCode": {
           "description": "If the action type is block, this field defines the default customer overridable http response status code.",
           "type": "integer",
-          "maximum": 599,
-          "exclusiveMaximum": false,
-          "minimum": 200,
-          "exclusiveMinimum": false
+          "enum": [
+            200,
+            403,
+            405,
+            406,
+            429
+          ]
```

Comment:
is this just a swagger fix and not a change to validation in an existing API?

Reply:
This is a fix that was already applied to a previous API version 2019-06-15 but hasn't been copied to API version 2020-03-31 which this was forked from. It's not a breaking change in our case because our service already returns an error for any responses other than these enums.